### DISCUSSION TITLE
Open the list from a record

### DISCUSSION THREAD

**Discussion by Walrand** [arnoudkooi/SN-Utils#224] [Ideas]
Hello,

Unless I did not find the correct slashcommand, this one does not exist and could be a nice addition to the plugin.

As a user on a form, I would like to input a slashcommand such as /list to open the list behind the record I am currently viewing.

Currently I either need to:
Adapt the URL from _x_testtesttest_test.do?sys_id=xxxxxxxxx_ to _x_testtesttest_test_list.do_
or 
Copy the table name and paste it in the application navigator and add .list

Thanks for reading
Have a good day

**Comment by arnoudkooi** [comment]
Currently no slashcommand, but this can be done via a ctrl-click in the header, which shows a link to the list view of the record

![clicktolist](https://user-images.githubusercontent.com/840781/166464718-5e5706ca-57d9-4788-9237-09ecbf6ad4d0.gif)

**Comment by Walrand** [comment]
This is as good as a slashcommand, thank you

**Reply by tdeniston** [reply]
I wrote a custom slash command to do something similar, but it creates a filter based on the current record. This could easily be adjusted just go to the table for the current record: 

Command: list
URL: $table_list.do?sysparm_query=sys_id=$sysid^OR$0
Hint: Move to list view with current record and encoded query

**Reply by arnoudkooi** [reply]
Perfect, thanks for stepping in Tim!

`/$table_list.do?sysparm_query=sys_id=$sysid^OR$0`

If you'd start the URL with a slash it forces it to open in a new tab.
In the current way, you'd hold CTRL-Enter or CTRL-Enter to force a new tab.
